panel/chat/interface.py

```diff
@@ -309,13 +314,18 @@ def _init_widgets(self):
     def _wrap_callbacks(
             self,
             callback: Callable | None = None,
-            post_callback: Callable | None = None
+            post_callback: Callable | None = None,
+            name: str = ""
         ):
         """
         Wrap the callback and post callback around the default callback.
         """
         def decorate(default_callback: Callable):
             def wrapper(self, event: param.parameterized.Event):
+                if name == "send" and not self.active_widget.value:
+                    # don't trigger if no message to prevent duplication
```

Comment:
This is primarily because value gets reset to "", and re-triggers the callback without this clause